dvc/params.yaml

```diff
@@ -76,50 +35,6 @@ models-and-features:
     model: xgb
     features-rd: image-tmd-rd
     features-proj: image-tmd-proj
-feature_extractors:
```

Comment:
Seems fine to remove. Also, notice that not even before this PR (see on branch `main`) these items were ever used in the `dvc.yaml`.